tss-esapi-sys/Cargo.toml

```diff
@@ -0,0 +1,20 @@
+[package]
+name = "tss-esapi-sys"
+version = "0.1.0"
+authors = ["Ionut Mihalcea <ionut.mihalcea@arm.com>",
+           "Hugues de Valon <hugues.devalon@arm.com>"]
+edition = "2018"
+description = "FFI wrapper around TSS 2.0 Enhanced System API"
+readme = "README.md"
+keywords = ["tpm", "tss", "esys", "esapi"]
+categories = ["api-bindings", "external-ffi-bindings", "cryptography"]
+license = "Apache-2.0"
+repository = "https://github.com/parallaxsecond/rust-tss-esapi"
+documentation = "https://docs.rs/crate/tss-esapi-sys"
+
```

Comment:
I think you need to add a `links` field, see [here](https://doc.rust-lang.org/cargo/reference/manifest.html#the-links-field)

Reply:
Uhm, and what happens if I need to `link` to multiple libraries? I think we're doing it wrong and should have one crate per library that we link with.......

Reply:
> I think we're doing it wrong and should have one crate per library that we link with.......
Good point, I don't know if ppl would make use of a `sys` crate + abstraction of those other libraries we use. But we can still do it in the future.